src/report_generator.cpp

```diff
@@ -774,10 +745,10 @@ This document is the Index by Status and Section for the <a href="lwg-active.htm
 
 
 void report_generator::make_sort_by_status_mod_date(std::vector<issue> & issues, fs::path const & filename) {
-   sort(issues.begin(), issues.end(), order_by_issue_number{});
-   stable_sort(issues.begin(), issues.end(), order_by_section{section_db});
-   stable_sort(issues.begin(), issues.end(), [](issue const & x, issue const & y) { return x.mod_date > y.mod_date; } );
-   stable_sort(issues.begin(), issues.end(), order_by_status{});
+   auto proj = [this](const auto& i) {
+      return std::make_tuple(lwg::get_status_priority(i.stat), ordered_date(i), ordered_section(section_db, i), i.num);
+   };
+   std::ranges::sort(issues.begin(), issues.end(), {}, proj);
```

Comment:
I can do this all day.
```suggestion
   std::ranges::sort(issues, {}, proj);
```